components/Favorite/Favorite.style.ts

```diff
@@ -0,0 +1,72 @@
+import styled from 'styled-components/native';
+
+import { DefaultText } from '../DefaultText/DefaultText.style';
+
+export const ModalContainer = styled.SafeAreaView`
+  background: #ffffff;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+`;
+
+export const ModalWrapper = styled.View`
+  flex: 1;
+  background: rgba(0, 0, 0, 0.4);
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+`;
+
+export const HeaderContainer = styled.View`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 8px 16px;
+`;
+
+export const OptionContainer = styled.TouchableOpacity`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 8px 16px;
+  border-top-width: 1px;
+  border-top-color: rgba(0, 0, 0, 0.05);
+`;
+
+export const HeaderText = styled(DefaultText)`
+  font-weight: 600;
+  font-size: 18px;
+  color: #888888;
+`;
+
+export const Button = styled.TouchableOpacity`
+  padding: 0 10px;
+`;
+export const ButtonText = styled(DefaultText)`
```

Comment:
Add a line space before this line

```suggestion

export const ButtonText = styled(DefaultText)`
```